### PR TITLE
Flat creation

### PR DESCRIPTION
**Pull request by Nicohelfen**

### CODE REVIEW

File: app/controllers/account/flats_controller.rb [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+    end
+
+    def edit
+      @flat = Flat.find(params[:id])
+    end
+
+    def update
+      @flat = Flat.find(params[:id])
+
+    whitelist = params.require(:flat).permit(:name, :description, :city, :zip_code, :street, :capacity, :price)
```
Comment (mouligno) [inline app/controllers/account/flats_controller.rb]: indentation

Comment (mouligno) [inline app/controllers/account/flats_controller.rb]: Should be in a private method

File: app/controllers/account/flats_controller.rb [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+
+    def new
+      @flat = Flat.new
+    end
+
+
+    def create
+
+      whitelist = params.require(:flat).permit(:name, :description, :city, :zip_code, :street, :capacity, :price )
```
Comment (mouligno) [inline app/controllers/account/flats_controller.rb]: Should be in private

File: app/views/account/flats/edit.html.erb [hunk 3]
```diff
@@ -0,0 +1,9 @@
+<h1>Update <%=@flat.name%> </h1>
```
Comment (mouligno) [inline app/views/account/flats/edit.html.erb]: Space between `<%=` and content

File: app/views/account/flats/index.html.erb [hunk 4]
```diff
@@ -0,0 +1,16 @@
+<%= link_to "Sign out", destroy_user_session_path, method: :delete %>
```
Comment (mouligno) [inline app/views/account/flats/index.html.erb]: Should be in navbar

File: app/views/account/flats/index.html.erb [hunk 5]
```diff
@@ -0,0 +1,16 @@
...
+
+<h1>Mes Flats</h1>
+
+<% @flats.each do |f| %>
+<ul class='flat-informations'>
```
Comment (mouligno) [inline app/views/account/flats/index.html.erb]: Indentation

File: app/views/account/flats/index.html.erb [hunk 6]
```diff
@@ -0,0 +1,16 @@
+<%= link_to "Sign out", destroy_user_session_path, method: :delete %>
+
+<h1>Mes Flats</h1>
+
+<% @flats.each do |f| %>
```
Comment (mouligno) [inline app/views/account/flats/index.html.erb]: Replace `f`by `flat` to be more explicit

File: app/views/account/flats/index.html.erb [hunk 7]
```diff
@@ -0,0 +1,16 @@
...
+<ul class='flat-informations'>
+  <li><%= link_to f.name,  flat_path(id: f.id) %></li>
```
Comment (mouligno) [inline app/views/account/flats/index.html.erb]: U don't need ;-)

File: app/views/flats/show.html.erb [hunk 8]
```diff
@@ -0,0 +1,10 @@
+<h1><%=@flat.name%></h1>
```
Comment (mouligno) [inline app/views/flats/show.html.erb]: Space missing between `<%= =>`

File: config/routes.rb [hunk 9]
```diff
@@ -1,11 +1,38 @@
 Rails.application.routes.draw do
+  root to: 'account/flats#index'
   devise_for :users
 
-  resources :flats, only: [:index, :show] do
+  resources :flats, only: [:index, :show ] do
```
Comment (mouligno) [inline config/routes.rb]: You don't use `index` in your controller

**Comment by mouligno** [comment]
Review done !

MRW i haven't seen any of the Star War movies and every one keeps asking me what i think about the new trailer

**Comment by Nicohelfen** [comment]
fixies done

**Comment by mouligno** [comment]
U can merge

**Comment by Nicohelfen** [comment]
merged

**Pull request merged**